pysc2/lib/features.py

```diff
@@ -283,6 +293,8 @@ def __init__(self,
       rgb_minimap_size: Sets rgb_minimap_width and rgb_minimap_height.
       rgb_minimap_width: The width of the rgb minimap observation.
       rgb_minimap_height: The height of the rgb minimap observation.
+      map_size: The map size.
```

Comment:
Please give better descriptions, eg:
      camera_width_world_units: The width of the feature layer camera in world
          units. This is needed for feature_units.
      map_size: The size of the map in world units, needed for feature_units.